packages/api/src/platforms/vtex/resolvers/validateCart.ts

```diff
@@ -20,17 +21,18 @@ import {
 
 type Indexed<T> = T & { index?: number }
 
-const getAttachments = (item: IStoreOffer) =>
-  item.itemOffered.additionalProperty?.filter(
-    (i) => i.valueReference === VALUE_REFERENCES.attachment
-  )
+const isAttachment = (value: IStorePropertyValue) =>
+  value.valueReference === VALUE_REFERENCES.attachment
 
 const getId = (item: IStoreOffer) =>
   [
     item.itemOffered.sku,
     item.seller.identifier,
     item.price,
-    item.itemOffered.additionalProperty?.map(getPropertyId).join('-'),
+    item.itemOffered.additionalProperty
+      ?.filter(isAttachment)
```

Comment:
Shouldn't this function be the same in the starters as well? How can we manage the fact that the starters can't know what attachments are?

Reply:
On the starters side I think it's ok. The issue arises when mixing checkout API with `@faststore/api`. If you are using only one or the other I think you're safe

Reply:
I see. So by that logic, we are able to insert VTEX concepts in the starters, as long as they follow `@faststore/api` schemas, right? That way we can guarantee they won't break if used with another platform (although it's not going to be a graceful experience).

Reply:
Yeah, I mean, we do that with the `channel`. Channel is a concept of `@faststore`, but what goes inside differs from each platform.

Reply:
got it!